katello_certs_tools/sslToolConfig.py

```diff
@@ -47,7 +47,7 @@
 SERVER_OPENSSL_CNF_NAME = 'katello-server-openssl.cnf'
 
 MD = 'sha256'
-CRYPTO = '-des3'
+CRYPTO = '-aes256'
```

Comment:
how does this affect existing installs? are they left untouched, are they upgraded, are they broken? ;)

Reply:
This is only used when generating the CA private key, so I think existing environments should be untouched. I can test this on a upgrade to be sure. I am not clear what happens with an existing des3 based key on a FIPS enabled EL8 environment but thats a whole 'nother can o' worms.

Reply:
For example, there is this guard in place https://github.com/Katello/katello-certs-tools/blob/master/katello_certs_tools/katello_ssl_tool.py#L156 and

Reply:
I think unbreaking already broken setups can be left for the future